### PR TITLE
Fix retirement tests

### PR DESCRIPTION
**Pull request by mfalesni**
They waited for a bad thing

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 33ed6b5b359b7c88bb4f5d1cc55fe2c323270da3:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e9d9c8a5e816d8e59df4595573ecc864b5df3508:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/cloud_infra_common/test_retirement.py [hunk 1]
```diff
@@ -41,20 +40,24 @@ def verify_retirement(vm):
...
-        wait_for(retirement_date_present, delay=30, num_sec=600, message="retirement_date_present")
+        # wait for the state to change
+        @pytest.wait_for(delay=15, timeout='6m')
+        def wait_for_retired():
+            vm.summary.reload()
+            return vm.summary.lifecycle.retirement_state.text_value.lower() == 'retired'
 
-        # wait for the power state to go to 'off'
-        wait_for(lambda: get_state() in {'off', 'suspended'}, delay=30, num_sec=360)
+        assert vm.summary.power_management.power_state.text_value in {'off', 'suspended', 'unknown'}
```
Comment (psav) [inline cfme/tests/cloud_infra_common/test_retirement.py]: Previously there was a wait for the power state to be poweroff, can we confirm that "retirement" only gets set on line 54, when the machine is powered off?

Comment (mfalesni) [inline cfme/tests/cloud_infra_common/test_retirement.py]: I would love to see some definition on what power state is expected after retirement. The previous code did not check the Retirement Status at all.

Comment (mfalesni) [inline cfme/tests/cloud_infra_common/test_retirement.py]: I guess this also depends on the underlying provider.

**Pull request merged**